nexus/tests/output/nexus_tags.txt

```diff
@@ -104,7 +104,31 @@ snapshot_delete                          DELETE   /v1/snapshots/{snapshot}
 snapshot_list                            GET      /v1/snapshots
 snapshot_view                            GET      /v1/snapshots/{snapshot}
 
-API operations found with tag "system"
+API operations found with tag "system/hardware"
+OPERATION ID                             METHOD   URL PATH
+networking_switch_port_apply_settings    POST     /v1/system/hardware/switch-port/{port}/settings
+networking_switch_port_clear_settings    DELETE   /v1/system/hardware/switch-port/{port}/settings
+networking_switch_port_list              GET      /v1/system/hardware/switch-port
```

Comment:
I know that these are under the `/v1/system/hardware...` path, but looking at the peers, I think this might make more sense under the "system/networking" tag (and possibly changing the path)

@internet-diglett and @rcgoodfellow what do you think?

Reply:
I could see this going either way. When we're applying/clearing switch port settings, we're doing so on a physical port which is a part of the system's hardware. These endpoints take selectors that specify the switch, but maybe the endpoint would be better as something like `/v1/system/hardware/switches/{switch}/switch-port/{port}/settings` now that #2996 has made switches UUID'd objects?

Reply:
@rcgoodfellow I'm working on wrapping that change up right now. So @ahl, no, the switch-ports _do_ need to be in the hardware section. I'm thinking about removing the `networking` prefix though.